### PR TITLE
Authentication Behaviors API beta documentation

### PR DESCRIPTION
**Pull request by medhir**
This PR adds beta API documentation for the new `authenticationBehaviors` property in the application object. 

This PR is tracked via the ADO work item 86934 in the Skilling ADO.

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -13202,6 +13202,32 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the `authenticationBehaviors` property to the [application](https://learn.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta) resource.",
+          "Target": "application"
+        }
+      ],
+      "Id": "5037c838-ee0a-4207-8969-620185db4240",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2023-04-20T01:47:34.2749095Z",
+      "WorkloadArea": "Identity and access",
```
Comment (medhir) [inline changelog/Microsoft.DirectoryServices.json:13229]: update to "Applications"

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -13202,6 +13202,32 @@
...
+      "SubArea": "Directory management"
```
Comment (medhir) [inline changelog/Microsoft.DirectoryServices.json:13230]: Update to ""

File: api-reference/beta/resources/application.md [hunk 3]
```diff
@@ -86,6 +86,7 @@ This resource supports:
 | appId | String | The unique identifier for the application that is assigned by Azure AD. Not nullable. Read-only. Supports `$filter` (`eq`). |
 |applicationTemplateId | String | Unique identifier of the applicationTemplate. Supports `$filter` (`eq`, `not`, `ne`).|
 | appRoles | [appRole](approle.md) collection | The collection of roles defined for the application. With [app role assignments](approleassignment.md), these roles can be assigned to users, groups, or service principals associated with other applications. Not nullable. |
+|authenticationBehaviors|[authenticationBehaviors](../resources/authenticationbehaviors.md)|**TODO: Add Description**|
```
Comment (medhir) [inline api-reference/beta/resources/application.md:89]: also add authenticationBehaviors to the json at the bottom of the page.

File: api-reference/beta/resources/authenticationbehaviors.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+# authenticationBehaviors resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Describes the authentication behaviors set in the context of an application. Authentication behaviors are boolean flags that provide applications flexibility in adopting breaking-change behaviors related to token issuance. These updated token issuance behaviors can be related to security mitigations, security improvements, or feature deprecations. 
+
+Applications can adopt new breaking changes by enabling a behavior (set the behavior to `true`), or continue using pre-existing behavior by disabling it (by setting the behavior to `false`). It should be noted that certain behaviors are not modifiable for an application in the interest of Microsoft maintaining a strong security posture.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/authenticationbehaviors.md:18]: Currently the last sentence isn't covered by the existing implementation. When new properties behave that way, you'll describe them so.
```suggestion
Applications can adopt new breaking changes by enabling a behavior (set the behavior to `true`), or continue using pre-existing behavior by disabling it (by setting the behavior to `false`).
```

File: api-reference/beta/resources/authenticationbehaviors.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+title: "authenticationBehaviors resource type"
+description: "Authentication behaviors are boolean flags that provide applications flexibility in adopting breaking-change behaviors related to token issuance. These updated token issuance behaviors can be related to security mitigations, security improvements, or feature deprecations."
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/authenticationbehaviors.md:3]: This metadata description has a limited char length; Let Line 16 ill the remaining gap.
```suggestion
description: "Authentication behaviors provide applications flexibility in adopting breaking-change behaviors related to token issuance."
```

File: api-reference/beta/resources/authenticationbehaviors.md (hunk 4 above)
Comment (FaithOmbongi) [inline api-reference/beta/resources/authenticationbehaviors.md:18]: Currently the last sentence isn't covered by the existing implementation. When new properties behave that way, you'll describe them so.
```suggestion
Applications can adopt new breaking changes by enabling a behavior (set the behavior to `true`), or continue using pre-existing behavior by disabling it (by setting the behavior to `false`).
```

File: api-reference/beta/resources/authenticationbehaviors.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+Describes the authentication behaviors set in the context of an application. Authentication behaviors are boolean flags that provide applications flexibility in adopting breaking-change behaviors related to token issuance. These updated token issuance behaviors can be related to security mitigations, security improvements, or feature deprecations. 
+
+Applications can adopt new breaking changes by enabling a behavior (set the behavior to `true`), or continue using pre-existing behavior by disabling it (by setting the behavior to `false`). It should be noted that certain behaviors are not modifiable for an application in the interest of Microsoft maintaining a strong security posture. 
+
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|removeUnverifiedEmailClaim|Boolean| Removes the `email` claim from tokens sent to an application when the email address's domain cannot be verified. |
+|requireClientServicePrincipal|Boolean| Requires a client application attempting to acquire a cross-tenant token from a resource application to have a Service Principal as a pre-requisite. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/authenticationbehaviors.md:25]:
```suggestion
|requireClientServicePrincipal|Boolean| Requires a client application attempting to acquire a cross-tenant token from a resource application to have a service principal as a pre-requisite. |
```

File: api-reference/beta/resources/application.md [hunk 7]
```diff
@@ -86,6 +86,7 @@ This resource supports:
 | appId | String | The unique identifier for the application that is assigned by Azure AD. Not nullable. Read-only. Supports `$filter` (`eq`). |
 |applicationTemplateId | String | Unique identifier of the applicationTemplate. Supports `$filter` (`eq`, `not`, `ne`).|
 | appRoles | [appRole](approle.md) collection | The collection of roles defined for the application. With [app role assignments](approleassignment.md), these roles can be assigned to users, groups, or service principals associated with other applications. Not nullable. |
+|authenticationBehaviors|[authenticationBehaviors](../resources/authenticationbehaviors.md)| The collection of authentication behaviors set for the application. Authentication behaviors are unset by default and must be explicitly enabled (or disabled) by setting a bheavior to `true` (or `false`). |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/application.md:89]:
```suggestion
|authenticationBehaviors|[authenticationBehaviors](../resources/authenticationbehaviors.md)| The collection of authentication behaviors set for the application. Authentication behaviors are unset by default and must be explicitly enabled (or disabled). |
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @medhir for your work on this PR. Left suggestions for fixes. Otherwise, approved with comments.

**Pull request merged**